src/main-view/content/row-settings/send-keys-row-settings.js

```diff
@@ -119,10 +155,8 @@ class SendKeysRowSettings extends Gtk.Grid {
     this.animationCombo.connect('changed', () => this.emit('changed'));
 
     // Layout
-    this.attach(modifiersLabel, 0, 0, 1, 1);
-    this.attach(this.modifiersEntry, 1, 0, 1, 1);
-    this.attach(keysLabel, 0, 1, 1, 1);
-    this.attach(this.keysEntry, 1, 1, 1, 1);
+    this.attach(shortcutLabel, 0, 0, 1, 1);
+    this.attach(this.shortcutButton, 1, 0, 1, 1);
```

Comment:
Now that you have removed one widget, you will need to adjust the position of the others, somethig like:

```js
    this.attach(repeatLabel, 0, 1, 1, 1);
    this.attach(this.repeatSwitc1h, 1, 1, 1, 1);
    this.repeatChanged(isRepeatActive);
    this.attach(animationLabel, 0, 3, 1, 1);
    this.attach(this.animationCombo, 1, 3, 1, 1);
    this.show_all();
  }
  repeatChanged(isRepeatActive) {
    this.remove_row(2);

    if (isRepeatActive) {
      this.attach(this.oppositeKeysLabel, 0, 2, 1, 1);
      this.attach(this.oppositeKeysEntry, 1, 2, 1, 1);
    } else {
      this.attach(this.onBeginEndLabel, 0, 2, 1, 1);
      this.attach(this.onBeginEndCombo, 1, 2, 1, 1);
    }

    this.show_all();
  }
```

Disclaimer: I noticed there is a bug in my code here, but to keep things simple and avoid conflicts, I'll fix it once this PR is merged :)